docker/android/android/util.py

```diff
@@ -0,0 +1,28 @@
+import re2 as re
+
+
+def windows(sequence, count):
+    for i in range(len(sequence) - count + 1):
+        yield sequence[i:i + count]
+
+
+def flatten(lst):
+    return [i for sublist in lst for i in sublist]
+
+
+def _is_match(pattern, string):
+    return re.search(pattern, string) is not None
+
+
+def is_test(string):
```

Comment:
Determining if something is a test isn't exactly trivial. Examples of tests include:
- `libgtest`
- `libgtest_main`
- `art-test-defaults`
- `cc_test`
- `cc_test_host`
- `py2-c-module_test_capi`.

While example of non-tests include:
- `latest`
- `fmtlib-non-test-defaults`.

So I've written a few false positives, and then used the fact that anything besides `gtest` if it is prefixed by a letter or digit is not a test. Any other printable character, however, is.